recipes/quantizer/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "quantizer" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/quantizer-{{ version }}.tar.gz
+  sha256: 1b685611b2fb31f032f918d8f288983e54186b6a097ae681004df8ed754109df
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
```

Comment:
When I look at the source code of this project, I can see that it doesn't define any run time requirements outside of python, but in the modules, it depends on `numpy`, `scipy`, and possibly others. It looks like version 0.2 fixes this definition, but if we know that this isn't going to run without those, they should be added.